### ISSUE TITLE
blue was quitted unexpectedly...

### ISSUE THREAD

**Issue by janjacobhofmann** [kunstmusik/blue#298]
By reducing the print out messages for the doppler opcode we managed to reduce the crashes of blue significantly so I could render without problems in deferred time. However it still crashes when it renders too dense structures in real time. The message displayed by the system after crashing then is (as usual):

"blue was quitted unexpectedly by using the  Plug-Ins libjvm.dylib"

Now, as I tried to render in better quality with ksmps = 5 instead of ksmps =  10 it also crashes when rendering to disk, but not right away but close to the end of the piece after several hours of rendering. The message displayed then is different and might give a trace what the reason for these crashes is:
 

"blue was quitted unexpectedly by using the  Plug-Ins libdoppler.dylib"

This applies to blue 2.5.14 with Csound 6.03 version 6.03.1 (double samples) May 12 2014, Mac OSX 10.7.5 with java JRE 1.7.0_67

**Comment by kunstmusik** [comment]
Hi @janjacobhofmann is this still an issue with latest versions of Csound and Java?

**Comment by janjacobhofmann** [comment]
Dear Steven,

good idea: I'll test it, but it might take two weeks until I can give you the results - I hope this is o.k.?

All the best,

Jan Jacob


sound         |         movement          |          object         |          space
sonic architecture       |        site: http://www.sonicarchitecture.de
spatial electronic composition  |  higher order ambisonic music

Am 14.04.2015 um 14:00 schrieb Steven Yi:

**Comment by kunstmusik** [comment]
I think it will take at least that long to get 2.6.0 done, so that
timeframe should work for me.  Thanks!

steven

**Comment by janjacobhofmann** [comment]
Great! I am also willing to do some testing with the beta of 2.6.0 but first I will test the current version of blue with the latest java and Csound to make it easier to trace the issues.

Best,

Jan Jacob


Am 14.04.2015 um 17:37 schrieb Steven Yi:

**Comment by janjacobhofmann** [comment]
Dear Steven,

the problem has vanished!!!!!!! I still can't believe it and hardly try to keep calm because I am sitting right now at the bar of the cinema where I do work. Still I feel like yelling and dancing:
blue right now renders my quite complicated latest piece "coloured dots" almost in real time WITHOUT CRASHING!!!!!!!!!! Also both cpus seem to be well balanced regarding their load!!!
Thanks to all the developers, which made that possible!

Now here is a more detailed report:
- I downloaded the latest release of JRE, Version 7 and installed it
- I downloaded Csound version 6.05 (double samples) Apr 25 2015
- I did *not* change my blue-install of 2.5.14

First, I got an error message I have never had before, because in one of my UDOs there was a line like

outs		aout1, aout2

as it should have been:

xout		aout1, aout2


Csound has never complained before, but with the new version it did and I quickly changed the line�and then it ran fine. (This UDO has not been used for regular compilation, still it is good to get an error now so I was able to change the mistake.)

THANKS A LOT, STEVEN!!!!

I will now download the latest test -version of blue and start some testing on it!

All the best,

Jan Jacob


sound         |         movement          |          object         |          space
sonic architecture       |        site: http://www.sonicarchitecture.de  
spatial electronic composition  |  higher order ambisonic music  


Am 14.04.2015 um 17:37 schrieb Steven Yi:
